string estimator_name
---
bool success
string message